events/index.html

```diff
@@ -41,39 +41,55 @@
 
 {% block content_main %}
 
-    {# TEMP The mocked event content is in the following paragraph #}
-    <p>
-        Events content goes here: {{vars.mock_post.content}}
-    </p>
+    <section class="media block block__flush-top block__sub">
+        <div class="media_body">
+            <h1>Upcoming events</h1>
```

Comment:
This is another existing pattern throughout the site. Worth considering but outside the scope of this PR.